Sources/PrimerSDK/Classes/User Interface/TokenizationViewModels/BankSelectorTokenizationProviding.swift

```diff
@@ -0,0 +1,30 @@
+//
+//  BankSelectorTokenizationProviding.swift
+//  PrimerSDK
+//
+//  Created by Alexandra Lovin on 16.11.2023.
+//
+
+import Foundation
+
+protocol TokenizationSetupAndCleaning {
+    func setup()
+    func cleanup()
+    func cancel()
+}
+
+protocol BankSelectorTokenizationProviding: TokenizationSetupAndCleaning {
```

Comment:
Nice 🙏 This paves the way for us to move away from the TokenizationViewModels. As I mention in the other comment, it would be great to attempt that already.